### ISSUE TITLE
[BUG] Last step of Deployment always fail the first time

### ISSUE THREAD

**Issue by TonyMasse** [logrhythm/EZ-Cloud#28]
**Describe the bug**
User reports that the last step of deployment always fail the first time

**To Reproduce**
1. Go to a Pipeline properties
2. Click on Add Deployment
3. Click the (+) button for a suitable Open Collector
4. Wait for the deployment to finish 
5. See error on the last step

**Expected behavior**
All the step should complete successfully.

**Screenshots**
![image](https://user-images.githubusercontent.com/50835088/211628255-3bb50409-2662-4ab9-ab31-61cf4ac185be.png)

**Desktop (please complete the following information):**
 - OS: Windows 2016
 - Browser: Chrome
 - Version: N/A

**Additional context**
The log source seems to be created correctly in the SIEM, yet the user claims that running the deployment a second time, the log source is created again (duplicate).
Issue seen on SIEM v7.9.

**Comment by TonyMasse** [comment]
Jira Issue ENG-23871

**Comment by TonyMasse** [comment]
SIEM 7.9 has a breaking change in the EMDB, to accommodate Axon LS types. This might be the cause of this issue.

**Comment by TonyMasse** [comment]
Root cause, on SIEM v7.9:

``` json
{"stillChecking":false,"errors":[{"message":"Could not find stored procedure 'LogRhythm_EMDB_HostIdentifierToMsgSource_Insert'.","code":"EREQUEST","number":2812,"state":62,"class":16,"serverName":"LRXM","procName":"OC_Admin_Upsert_Log_Source_Virtualisation_To_OpenCollector_LogSource","lineNumber":766}],"outputs":["69 row(s) returned"],"payload":[{"UpdatedOpenCollectorLogSourceID":80,"UpsertedVirtualLogSourceID":82,"NewCommentID":50}]}
```

**Comment by TonyMasse** [comment]
Fixed with PR #29
